appveyor.yml

```diff
@@ -1,4 +1,4 @@
-image: Visual Studio 2019
+image: Visual Studio 2022
```

Comment:
This is to allow https://github.com/S7NetPlus/s7netplus/blob/5d3f01e59e61720b3dedd3342b433824b6287214/S7.Net/PlcAsynchronous.cs#L52